test/topology/test_null.py

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+from cassandra.protocol import InvalidRequest   # type: ignore
+from pylib.util import unique_name
+import pytest
+
+
+@pytest.fixture(scope="module")
+def table1(cql, test_keyspace):
+    table = test_keyspace + "." + unique_name()
+    cql.execute(f"CREATE TABLE {table} (p text, c text, v text, primary key (p, c))")
+    yield table
+    cql.execute("DROP TABLE " + table)
+
+
+def test_delete_empty_string_key(cql, table1):
```

Comment:
I'm curious what this test is about... How is empty string test (which it seems you copied from cql-pytest?) related to the topology?
If is just a placeholder (?) and will be relaced later, or something, please explain in the commit message.

Reply:
This test is about nothing. I wrote about it before. It tests that clusters start and stop correctly.

Reply:
In other words, it tests suite.yaml